docs/modules/ROOT/pages/configuration.adoc

```diff
@@ -114,6 +114,13 @@ inherit_from:
   - ../conf/.rubocop.yml
 ----
 
+`inherit_from` also accepts a glob, for example:
```

Comment:
You might also want to say a few words about potential use-cases here.

Reply:
Added some more text about using this to manage per-component `.rubocop_todo.yml` files. I'm not actually sure about other use cases since that use-case was the main motivator here 🤔 Let me know what ya think!

Reply:
Looks good to me.